front/src/modules/auth/graphql/mutations/generateShortTermToken.ts

```diff
@@ -0,0 +1,11 @@
+import { gql } from '@apollo/client';
+
+export const GENERATE_ONE_SHORT_TERM_TOKEN = gql`
```

Comment:
Could we change the naming to be Transient Token
It's kind of duplicated with the Login Token but I like to have both separated for now.

So we have:
- TransientToken that can be generated with an accessToken (payload containing workspaceMemberId + workspaceId, we could actually add userId to)
- LoginToken that can be generated by any login method (payload containing email)